prow/cmd/deck/static/pr.html

```diff
@@ -76,9 +76,21 @@ <h4 class="mdl-dialog__title"></h4>
 <dialog id="query-dialog" class="pr-status-dialog mdl-dialog">
     <h4 class="mdl-dialog__title">Query Details</h4>
     <div class="mdl-dialog__content">
-        <p class="query-details-section">All required labels</p>
+        <div id="query-detail-milestone" class="hidden">
+            <p class="detail-title">Milestone</p>
+            <p class="detail-data"></p>
+        </div>
+        <div id="query-detail-exclude" class="hidden">
+            <p class="detail-title">Exclude branches</p>
+            <p class="detail-data"></p>
+        </div>
+        <div id="query-detail-include" class="hidden">
+            <p class="detail-title">Include branches</p>
```

Comment:
possibly "included" and "excluded" for these? to be consistent with required and forbidden

Reply:
done +1